config/defaults.go

```diff
@@ -38,6 +38,8 @@ const (
 	DefaultNoBootstrap = false
 )
 
+var DefaultGenesisPath = os.Getenv("PWD") + "../genesis.json"
```

Comment:
this should be `CWD`, but it should really just be `./genesis.json`

Reply:
what's the difference between PWD and CWD? i don't think it'll work with `./genesis.json` since this file is inside `config`

Reply:
changed it to just `./genesis.json`, idk why I didn't do that in the first place lol

Reply:
`pwd` => print working directory, `CWD` => env var for current working directory :heart: